adapters/resetdigital/resetdigitaltest/examplary/simple-banner.json

```diff
@@ -0,0 +1,88 @@
+{
+  "mockBidRequest": {
+    "id": "12345",
+    "imp": [
```

Comment:
should update or add json test:

- to have request with multiple imps
- to have multi-format imp request

Reply:
By standard our solution works with single impressions. Should we adapt this? Or add a test to it to do it that way?

Reply:
So the solution here is to make it explicit on the docs that is single impression on docs? What should be the doc to use @onkarvhanumante ?

Reply:
> So the solution here is to make it explicit on the docs that is single impression on docs? What should be the doc to use @onkarvhanumante ?

Raise docs PR. Refer https://github.com/prebid/prebid.github.io/pull/5331 as example